packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -150,8 +150,8 @@ class DiagramFlutterBinding extends BindingBase
     with
         GestureBinding,
         SemanticsBinding,
-        ServicesBinding,
         SchedulerBinding,
+        ServicesBinding,
```

Comment:
This change doesn't seem necessary.

Reply:
I know this was something I wanted to mention in what I learned this week. But services binding is a mixin on BindingBase and SchedularBinding. So for some reason I needed to put the SchedularBinding mixin first before the ServicesBinding mixin. If you look at the other mixins below ServicesBinding those also extend multiple classes.